client/my-sites/plans-v2/utils.ts

```diff
@@ -3,11 +3,12 @@
  */
 import { translate, TranslateResult } from 'i18n-calypso';
 import { get } from 'lodash';
+import moment from 'moment';
```

Comment:
Is this a legit way to import `moment` in here? In components, the recommended way is to use `useLocalizedMoment`.

Reply:
I would personally pass in `moment` as an argument to the function below.